lang/src/org/partiql/lang/ast/ExprNodeToStatement.kt

```diff
@@ -151,83 +148,87 @@ fun ExprNode.toAstExpr(): partiql_ast.expr {
 
             // These are handled by `toAstDml()`
             is DataManipulation, is CreateTable, is CreateIndex, is DropTable, is DropIndex ->
-                error("Can't transform ${node.javaClass} to a partiql_ast.expr }")
+                error("Can't transform ${node.javaClass} to a PartiqlAst.expr }")
         }
     }
 }
 
-private fun GroupBy.toAstGroupSpec(): group_by =
-    group_by(
-        this.grouping.toAstGroupStrategy(),
-        partiql_ast.group_key_list(this.groupByItems.map { partiql_ast.group_key(it.expr.toAstExpr(), it.asName?.name?.asPrimitive()) }),
-        this.groupName?.name?.asPrimitive())
+private fun GroupBy.toAstGroupSpec(): PartiqlAst.GroupBy =
+    PartiqlAst.build {
+        groupBy_(
```

Comment:
Is the `_` part of the name?

Reply:
Yes, added a comment to the generated code to explain it.